contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,141 @@
+
+Enter EVELYN. Alexey knew they needed a dollar estimate of the impact.
+
+"Instead of building the entire backend, we created a simple [fake door test](/tutorials/fake-door-test) on the frontend. Users could select their preferred tier – but at checkout, we let them know they were being upgraded to the best tier and at the cheapest price. In reality, this was actually the existing one-size-fits-all pricing.
+
+Now we knew how much money the company stood to make! We prioritized it immediately, and it was incredibly easy to ask the rest of the company to work on it for a few months."
+
+## Problem 3: Your experiment win size is low
+
+Your growth team has been around for a few quarters and you've picked all the low-hanging fruit. Your new experiments require more effort AND have less impact. Now what?
+
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Increase the number of high-quality project ideas.
+2. Make your engineers own their experiments.
+
+### Increase the number of high-quality project ideas
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVENLYN framework mentioned above, you can distill the list to only the best ideas.
+
+> **💡 Tip:** To create an inclusive ideas culture, make your [brainstorms asynchronous](https://source.opennews.org/articles/redesigning-brainstorming-asynchronous/).
+
+### Make your engineers own their experiments
+
+Every engineer must be responsible for making their experiments as impactful as possible – even if it wasn't originally their idea. 
+
+Empowerment is the key here. When engineers take charge of their experiments, they become more invested in the outcome, leading to better results.
+
+In practice, this means:
+
+- Engineers work to improve the initial hypothesis.
+- Engineers develop further iterations to test. 
+- Removing processes that prevent engineers from making quick decisions on their own.
+- Allowing engineers space to fail without being harsh on them.
+
+## Problem 4: Your team is afraid of making mistakes
+
+If your team is not making mistakes, you're not moving fast enough. 
+
+In fact, it's not uncommon for growth teams to cause a few incidents! And their goal must not be to avoid them.
+
+"When airbags were [introduced](https://www.purdue.edu/uns/html4ever/2006/060927ManneringOffset.html) in cars, the average speed on the highway went up. This is essentially what you want to do for your growth team. Give them the freedom to make mistakes, but make sure they resolve them quickly."
```

Comment:
wait, the cool thing about this anecdote is that the crash rate didn't go down! Don't miss that part.

Reply:
Hmm not sure we need to include this to make the point 🙈